src/QuarkChainNameService.js

```diff
@@ -1,18 +1,31 @@
 "use strict"
+import {
+  registryInit,
+  getResolver,
+} from "./helper/qcns/registryService";
+import {
+  resolverInit,
+  getAddress
+} from "./helper/qcns/resolverService";
 
 class QuarkChainNameService {
   // Provider URL
-  constructor(restURL) {
-    this.restURL = restURL
+  constructor(restURL, networkId) {
+    this.restURL = restURL;
+    this.networkId = networkId;
   }
 
   async getResolver(name) {
     // TODO using web3 to fetch data
-    console.log(name);
+    registryInit(this.restURL, this.networkId);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
   }
 
-  async getAddress(name) {
-    
+  async getAddress(name, resolverAddr) {
```

Comment:
This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`